migrator/migrations/m_192_to_m_193_report_config_migrations/migration_test.go

```diff
@@ -0,0 +1,82 @@
+//go:build sql_integration
+
+package m192tom193
+
+import (
+	"context"
+	"testing"
+
+	reportConfigDS "github.com/stackrox/rox/central/reports/config/datastore"
```

Comment:
We should not use the datastore in a migration test.  In fact I'm surprised the linters allowed it.  The datastore could change in such ways down the line that could impact the ability of this test to run.